.ci/docs/DebugCoreDumps.md

```
@@ -9,16 +9,16 @@ load it and run `gdb` against the crash.
 The export path is printed in the Jenkins build log under the step:
 
 ```
-exportEnroot: export 'nixl-ci-<ucx_version>-<BUILD_NUMBER>' -> <path>
+exportEnroot: export 'nixl-ci-<BUILD_NUMBER>' -> <path>
 ```
 
 Files follow this naming convention (the sqsh is named after the container):
 
 ```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)